apps/server/src/modules/pseudonym/repo/pseudonyms.repo.spec.ts

```diff
@@ -118,6 +118,42 @@ describe('PseudonymRepo', () => {
 		});
 	});
 
+	describe('findPseudonymsByUserId', () => {
+		describe('when pseudonym is existing', () => {
+			const setup = async () => {
+				const user = userFactory.buildWithId();
+				const pseudonym1: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user.id });
+				const pseudonym2: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user.id });
+
+				await em.persistAndFlush([pseudonym1, pseudonym2]);
+
+				return {
+					user,
+					pseudonym1,
+					pseudonym2,
+				};
+			};
```

Comment:
Could you please also insert some test pseudonyms not related to the given userId to be sure you're only finding the ones for sure related to the given userId?